### PR TITLE
add scikit-downscale

### PR DESCRIPTION
**Pull request by jhamman**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by jhamman** [comment]
@jukent and @andersy005 can you confirm here that you're okay being listed as maintainers.

**Comment by andersy005** [comment]
👍 for being a maintainer

**Comment by jukent** [comment]
Same

**Comment by jhamman** [comment]
@conda-forge-admin - this is ready to fly!

File: recipes/scikit-downscale/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+    - setuptools_scm
+  run:
+    - python >=3.6
+    - xarray
```
Comment (synapticarbors) [inline recipes/scikit-downscale/meta.yaml:24]: My only question is if we need to reproduce the version restrictions on `xrray` and `scikit-learn` that are in the `requirements.txt` file upstream:

https://github.com/jhamman/scikit-downscale/blob/master/requirements.txt#L1-L2

Comment (jhamman) [reply recipes/scikit-downscale/meta.yaml]: Thanks for the review @synapticarbors. I went ahead and added the version restrictions.

**Comment by jhamman** [comment]
Thanks @synapticarbors!

**Pull request merged**